app/controllers/application_controller.rb

```diff
@@ -12,6 +12,16 @@ def switch_language
     redirect_back(fallback_location: root_path)
   end
 
+  def error_server
+    flash[:danger] = t "server.error_message"
+    redirect_to root_path
+  end
+
+  def user_not_found
+    flash[:danger] = t "not_found.user"
```

Comment:
tự nhiên thấy `.user` cái thấy nó sai sai, nếu đã xác định tạo method dùng chung thì chỉ nên là find_by_id_not_found
và mess chỉ cần record not found chẳng hạn